pr-data.csv

```diff
@@ -2030,7 +2030,8 @@ https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,or
 https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.TestStatsSerde.testLazyBinarySerDe,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/cli,org.apache.ignite.cli.commands.cliconfig.CliConfigSubCommandTest.noKey,ID,Accepted,https://github.com/apache/ignite-3/pull/888,The PR was closed but the commit https://github.com/apache/ignite-3/commit/81851d38f6197ea414552148d9db372c09b43eb8 was added to the main branch of the repository
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/cli,org.apache.ignite.cli.deprecated.IgniteCliInterfaceTest$Cluster.initSuccess,ID,Accepted,https://github.com/apache/ignite-3/pull/888,The PR was closed but the commit https://github.com/apache/ignite-3/commit/81851d38f6197ea414552148d9db372c09b43eb8 was added to the main branch of the repository
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/configuration,org.apache.ignite.internal.configuration.asm.ConfigurationAsmGeneratorTest.testConstructInternalConfig,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/configuration,org.apache.ignite.internal.configuration.asm.ConfigurationAsmGeneratorTest.testConstructInternalConfig,ID,Deleted,,https://github.com/apache/ignite-3/commit/960ce0a72dc8486bca6458f82471dc6d35265482
```

Comment:
Shouldn't this have a different class name? Also, is the test really being `Deleted` or just renamed? Could you run NonDex in `960ce0a72dc8486bca6458f82471dc6d35265482` to check if the test is still `ID`?

Reply:
I have added the correct class name. I can add the test in gradle file if it is flaky in the next PR. @darko-marinov